startup/00-base.py

```diff
@@ -64,3 +64,8 @@ def wait_for_connection(self, timeout=DEFAULT_CONNECTION_TIMEOUT):
 
 RE.md["beamline_name"] = "NYX"
 RE.md["facility"] = "NSLS-II"
+
+# TODO: comment out when in production.
```

Comment:
Do we want to include this in the main code? If nothing else, it would probably be good to put it in an "if" block with an environment variable toggling it.